tests/e2e_test.go

```diff
@@ -0,0 +1,419 @@
+							},
+						},
+					},
+				},
+			},
+			DestinationSubnets: []config.DestinationSubnet{
+				{
+					SubnetID:          subnetB.String(),
+					ChainID:           blockchainIDB.String(),
+					VM:                config.EVM.String(),
+					EncryptConnection: false,
+					APINodeHost:       hostB,
+					APINodePort:       portB,
+					AccountPrivateKey: fundedKeyStr,
+				},
+			},
+		}
+
+		data, err := json.MarshalIndent(relayerConfig, "", "\t")
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		f, err := os.CreateTemp(os.TempDir(), "relayer-config.json")
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		_, err = f.Write(data)
+		gomega.Expect(err).Should(gomega.BeNil())
+		relayerConfigPath = f.Name()
+
+		log.Info("Created awm-relayer config", "configPath", relayerConfigPath, "config", string(data))
+	})
+
+	ginkgo.It("Build Relayer", ginkgo.Label("Relayer", "Build Relayer"), func() {
+		// Build the awm-relayer binary
+		cmd := exec.Command("./scripts/build.sh")
+		out, err := cmd.CombinedOutput()
+		fmt.Println(string(out))
+		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	// Send a transaction to Subnet A to issue a Warp Message from the Teleporter contract to Subnet B
+	ginkgo.It("Send Message from A to B", ginkgo.Label("Warp", "SendWarp"), func() {
+		ctx := context.Background()
+
+		// Create a channel to communicate with the goroutine
+		cmdOutput := make(chan string)
+
+		// Run awm relayer binary with config path
+		relayerContext, relayerCancel := context.WithCancel(ctx)
+		cmd := exec.CommandContext(relayerContext, "./build/awm-relayer", "--config-file", relayerConfigPath)
+
+		// Set up a pipe to capture the command's output
+		cmdReader, _ := cmd.StdoutPipe()
+
+		// Start the command
+		err := cmd.Start()
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Start a goroutine to read and output the command's stdout
+		go func() {
+			scanner := bufio.NewScanner(cmdReader)
+			for scanner.Scan() {
+				log.Info(scanner.Text())
+			}
+			cmdOutput <- "Command execution finished"
+		}()
+
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		nonceA, err := chainAWSClient.NonceAt(ctx, fundedAddress, nil)
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		nonceB, err := chainBWSClient.NonceAt(ctx, fundedAddress, nil)
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		log.Info("Packing teleporter message", "nonceA", nonceA, "nonceB", nonceB)
+
+		payload, err = teleporter.PackTeleporterMessage(common.Hash(blockchainIDB), teleporterMessage)
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		packedInput, err := warp.PackSendWarpMessage(warp.SendWarpMessageInput{
+			DestinationChainID: common.Hash(blockchainIDB),
+			DestinationAddress: teleporterContractAddress,
+			Payload:            payload,
+		})
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Send a transaction that simulates the Teleporter contract calling sendWarpMessage with a Teleporter message payload
+		tx := types.NewTx(&types.DynamicFeeTx{
```

Comment:
Similar to the above comment about relayer config, it might be nice to have a default teleporter message at global scope. Ideally relayer e2e tests shouldn't depend on teleporter message content, as that would be better suited for the teleporter integration tests. There are probably exceptions of course.